frontend/src/components/charts/Indicator.js

```diff
@@ -7,6 +7,34 @@ import moment from 'moment';
 class Indicator extends Component {
   constructor(props) {
     super(props);
+    this.state = { localComputedTimestamp: null };
+  }
+
+  /**
+   * Updates the local computed timestamp with the one received from the props. Case when it comes from a query via xhr or via WS
+   * @param {props} param0
+   * @returns
+   */
+  static getDerivedStateFromProps({ data: { computedTimestamp } }) {
+    return { localComputedTimestamp: computedTimestamp };
+  }
+
+  /**
+   * @summary sets the local timestamp + 1 second, called on one second interval after the component mounts
+   */
+  updateTimestamp = () => {
+    const { localComputedTimestamp } = this.state;
+    const newTime = moment(localComputedTimestamp)
+      .add(1, 'seconds')
+      .format('hh:mm A');
+    return this.setState({ localComputedTimestamp: newTime });
+  };
+
+  /**
+   * @summary lifecycle in which we are calling the funtion that updates the local timestamp on one second
+   */
+  componentDidMount() {
+    this.interval = setInterval(() => this.updateTimestamp(), 1000);
```

Comment:
IMHO updating each minute is enough.
1sec it's too eager and brings no value for the user.

Reply:
Ok I can change that